tools/ci-build.sh

```diff
@@ -85,8 +88,13 @@ sudo apt-get update
 sudo apt-get install -y --no-install-recommends ${P4C_DEPS}
 sudo pip3 install --upgrade pip
 sudo pip3 install -r ${P4C_DIR}/requirements.txt
-# ! ------  END CORE -----------------------------------------------
 
+if ! command -v ninja &> /dev/null
+then
+    echo "ninja could not be found, using 'make' instead"
+    BUILD_GENERATOR="Unix Makefiles"
+fi
```

Comment:
I wonder if this is a good idea. I suggest we fail in CI if Ninja is not available. That will make sure we are really building with Ninja and not accidentally building with make because Ninja got lost somewhere. The check would be there only if Ninja is selected.